andrewinsoul/myDiary-python
# chore(database-setup): Setup tables and relationships

**What does this PR do**
This PR configures the database connection details and defines the models with their corresponding relationship

**Description of Tasks completed**
- Add the database connection details
- Add the application's models and its corresponding relationship

**How should this be manually tested**
- Fetch the branch locally by running `git fetch origin ch-setup-database-166233642`
- Checkout to the branch by running `git checkout ch-setup-database-166233642`
- Make sure you are on the virtual env and run `python manage.py migrate`
- The tables will be created with the corresponding relationships

**What are the relevant Pivotal Tracker Stories**
[166233642](https://www.pivotaltracker.com/story/show/166233642)
